src/main/resources/application.conf

```diff
@@ -0,0 +1 @@
+loggerLevel="INFO"
```

Comment:
that's still rather a not optimal way:
check this out: https://github.com/benblack86/finest